spk/usbserial/Makefile

```diff
@@ -1,21 +1,23 @@
 SPK_NAME = UsbSerialDrivers
 SPK_VERS = 6
-SPK_REV = 7
+SPK_REV = 1
```

Comment:
initial packages should start with (package-) revision 1

maybe for `SPK_VERS` we start at 1 too (or use 7 or 7.00 or 7.0.0 since usbutils is version 700)
or is there any other version of an included component available?